scuba/config.py

```diff
@@ -73,20 +73,22 @@ def from_yaml(self, node):
 
 
 def find_config():
-    '''Search up the diretcory hierarchy for .scuba.yml
+    '''Search up the directory hierarchy for .scuba.yml
 
-    Returns: path, rel on success, or None if not found
+    Returns: path, rel, scuba_config on success, or None if not found
         path    The absolute path of the directory where .scuba.yml was found
         rel     The relative path from the directory where .scuba.yml was found
                 to the current directory
+        config  The loaded configuration
     '''
     cross_fs = 'SCUBA_DISCOVERY_ACROSS_FILESYSTEM' in os.environ
     path = os.getcwd()
 
     rel = ''
     while True:
-        if os.path.exists(os.path.join(path, SCUBA_YML)):
-            return path, rel
+        config = os.path.join(path, SCUBA_YML)
+        if os.path.exists(config):
+            return path, rel, load_config(config)
```

Comment:
`config` might be a misleading name since we identified that as a "named" return value. How about:
```suggestion
        cfg_path = os.path.join(path, SCUBA_YML)
        if os.path.exists(cfg_path):
            return path, rel, load_config(cfg_path)
```

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db